ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -163,6 +163,18 @@ def test_update_datatokens(
     assert _asset.datatokens[0].get("address") == old_datatokens[0].get("address")
     assert _asset.services[0].datatoken == old_datatokens[0].get("address")
 
+    nft_token = publisher_ocean_instance.get_nft_token(_asset.nft["address"])
+    bn = publisher_ocean_instance.web3.eth.block_number
+    updated_event = nft_token.get_event_log(
+        event_name="MetadataUpdated", from_block=bn, to_block=bn, filters=None
+    )[0]
+    assert updated_event.args.updatedBy == publisher_wallet.address
+
+    validation_event = nft_token.get_event_log(
```

Comment:
as discussed @mariacarmina please make sure these events are simultaneous, not just that they exist separately from one another